### PR TITLE
i18n(ja): Update components.mdx

### PR DESCRIPTION
**Pull request by morinokami**
#### Description

- Update components.mdx.
  - [diff](https://github.com/withastro/starlight/commits/main/docs/src/content/docs/guides/components.mdx?since=2024-02-18T14:09:58.000Z)
    - https://github.com/withastro/starlight/pull/1568
    - https://github.com/withastro/starlight/pull/1308
    - https://github.com/withastro/starlight/pull/1564

### CODE REVIEW

File: docs/src/content/docs/ja/guides/components.mdx [hunk 1]
```diff
@@ -59,16 +59,24 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 <Tabs>
-	<TabItem label="恒星">シリウス、ベガ、ペテルギウス</TabItem>
-	<TabItem label="衛星">イオ、エウロパ、ガニメデ</TabItem>
+	<TabItem label="恒星" icon="star">
```
Comment (HiDeoo) [inline docs/src/content/docs/ja/guides/components.mdx:62]: I cannot add a comment to the line for the description above the example but I think the description is missing the new additional line added to the description:

> Use the optional `icon` attribute to include one of [Starlight’s built-in icons](#all-icons) next to the label.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view).

I think 1 sentence is missing in the translation but everything else looks perfect to me :thumbsup:

File: docs/src/content/docs/ja/guides/components.mdx (hunk 1 above)
Comment (morinokami) [reply docs/src/content/docs/ja/guides/components.mdx:62]: Thanks! I've added the missing sentence 🙏

**Review by HiDeoo — APPROVED** [review]
Thanks for the quick fix!

**Pull request merged**